user_notification/tests.py

```diff
@@ -24,3 +26,34 @@ def test_exception(n, x):
         JobAlert.calc_check_date(n, x)
     assert str(
         excinfo.value) == 'frequency_in_days should be Greater than or equal to 1'
+
+
+# Tests to check if alest details are correct
+
+@pytest.mark.parametrize('n,expected', [
+    ("Yosi", "<QuerySet [<JobAlert: Yosi>]>"),
+    ("Tami", "<QuerySet [<JobAlert: Tami>]>"),
+    ("Itay", "Job Alert Not Exist"),
+])
+@pytest.mark.django_db
+def test_check_if_alert_exist(n, expected):
+    insertDataToDB()
+    assert str(JobAlert.check_if_alert_exist(n)) == expected
+    deleteAllobjects()
+
+
+def deleteAllobjects():
+    JobAlert.objects.all().delete()
+    UserAccount.objects.all().delete()
+    User.objects.all().delete()
+
```

Comment:
make sure that you delete each row and not all the database with .all().delete()
@Varditli